index.html

```diff
@@ -55,6 +55,66 @@ <h2>Filtres</h2>
           cette ville
         </p>
       </section>
+      <!--section : the most popular-->
+      <section>
+        <div id="most_popular">
+          <!--mp heading-->
+          <div id="most_popular_heading">
+            <h2>Les plus populaires</h2>
+            <i class="fas fa-chart-line"></i>
+          </div>
+          <!--mp hotel 1-->
+          <a href="#" class="hotel_card">
+            <img
+              class="hotel_photo"
+              src="./assets/img/emile-guillemot-Bj_rcSC5XfE-unsplash.jpg"
+              alt="chambre hôtel Le soleil du matin"
```

Comment:
Alt must be descritive

Reply:
ok alt description changed, tried to be more "insightful" so tell me if it's ok for you.